content/tutorials/webrtc/infrastructure/en/index.html

```diff
@@ -496,6 +496,7 @@ <h2 id="after-signaling-using-ice-to-cope-with-nats-and-firewalls">After signali
     }
   ]
 }</pre>
+<p><b>Note:</b> the TURN credentials shown above are time-limited and have expired in September 2013. TURN servers are expensive to run and you wіll need to pay for your own servers or find a service provider. To test credentials you can use the <a href="https://webrtc.github.io/samples/src/content/peerconnection/trickle-ice/">candidate gathering sample</a> and check if you get a candidate with type relay.</p>
```

Comment:
Maybe slightly rephrase: 'the TURN credentials example shown above was time-limited and expired in September 2013'

Also I think  '... type relay' should have 'relay' in italics.